docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,286 @@
+
+
+Using remove
+-----------------
+
+**removeRegularExpressionKey**
+
+This replaces all matches of regular expression with “”, write as
+
+.. code-block::
+
+  removeRegularExpressionKey1.key=\\d
+
+all digits will be removed from the result.
+
+**removeKeysAndContentBetweenKeys**
+
+This removes all characters found between key1 and key2, including key1 and key2. By writing
+
+.. code-block::
+
+  removeKeysAndContentBetweenKeys1.key1=<RecordID>
+  removeKeysAndContentBetweenKeys1.key2=</RecordID>
+
+“<RecordID> 12</RecordID>” will be removed from the result.
+
+**removeKey**
+
+This removes all matching occurrences of provided key, by writing
+
+.. code-block::
+
+  removeKey1.key=RecordID
+
+all 4 “RecordID” will be removed from the result.
+
+
+Others
+-----------------------
+
+**decodeUnzipContentBetweenKeys**
+
+This decodes and unzips file content between key1 and key2, if replaceNewlines is true, it will replace all “\\r” with "[CARRIAGE RETURN]" and all "\\n" with "[LINE FEED]". In the example, there is an element ``<File>UEsDB...</File>``, it is an encoded zip file named "myZipfile.zip", which contains a text file "test.txt" with content "Larva Test" written in it. By writing
+
+.. code-block::
+
+  decodeUnzipContentBetweenKeys1.key1=<File>
+  decodeUnzipContentBetweenKeys1.key2=</File>
+  decodeUnzipContentBetweenKeys1.replaceNewlines=false
+
+it will first decode the content and then unzip this zip file to format “<tt:file xmlns:tt=\"testtool\"><tt:name>myZipfile/test.txt</tt:name><tt:content>Larva Test</tt:content></tt:file>”, and put it in between the 2 keys.
+
+**canonicaliseFilePathContentBetweenKeys**
+
+This canonicalizes file path content between key1 and key2. In the example, there is an element ``<FilePath>C:\Users\..\myZipfile.zip</FilePath>``, by writing
+
+.. code-block::
+
+  canonicaliseFilePathContentBetweenKeys1.key1=<FilePath>
+  canonicaliseFilePathContentBetweenKeys1.key2=</FilePath>
+
+it replaces “C:\\Users\\..\\myZipfile.zip” with the canonical pathname of the file object “C:\\myZipfile.zip”.
+
+**formatDecimalContentBetweenKeys**
+
+This formats decimal content between key1 and key2, the goal of using it is to be able to compare strings by formatting a decimal number to a canonical representation. For an integer, it will be presented as a string of digits, for example, “100” will still be “100”. And for a number which has a decimal fraction, it will be presented as a string, because the decimal precision is not known, for example, “003.0100” will become “3.01”. In the example, there is an element ``<Decimal>12.34</Decimal>``, by writing
+
+.. code-block::
+
+  formatDecimalContentBetweenKeys1.key1=<Decimal>
+  formatDecimalContentBetweenKeys1.key2=</Decimal>
+
+it will format this numeric value to ``<Decimal>12.34</Decimal>``, in this case nothing has changed.
+
+
+To make the example result at the beginning of this subsection pass the Larva test, you can write following ignores in your test: 
+
```

Comment:
You are no longer writing here about "formatDecimalContentBetweenKeys". Can you add a heading to make this clear?

Reply:
This issue has been resolved. Thanks.